### PR TITLE
KAZOO-5751: more client fixes for pivot/conferences

### PR DESCRIPTION
**Pull request by jamesaimonetti**

### CODE REVIEW

File: core/kazoo_documents/src/kzd_conference.erl [hunk 1]
```diff
@@ -0,0 +1,31 @@
+-module(kzd_conference).
```
Comment (lazedo) [inline core/kazoo_documents/src/kzd_conference.erl]: @jamesaimonetti can you check the `cps` branch ? there's a module there's a `kzd_conference` module

Comment (jamesaimonetti) [reply core/kazoo_documents/src/kzd_conference.erl]: @lazedo i guess we'll race on getting @k-anderson to merge one or the other first! :)

**Comment by jamesaimonetti** [comment]
@lazedo since my kzd_conference is much smaller, want to merge this and rebase cps with it ? :)

**Comment by lazedo** [comment]
@jamesaimonetti not sure, this kzd_conference is document oriented and the one in cps is event oriented. i'm ok with renaming the one in cps to something more suitable. thoughts ?

**Comment by jamesaimonetti** [comment]
@lazedo we have kz_call_event, could yours be kz_conference_event?

**Comment by lazedo** [comment]
@jamesaimonetti sold!

**Pull request merged**